### PR TITLE
Fix mono package build versions

### PR DESCRIPTION
**Pull request by ross-pfahler**
Fixes build issue with 
```
error Couldn't find package "@react-types/typography@^3.0.0-alpha.1" required by "@react-spectrum/typography@^3.0.0-alpha.1" on the "npm" registry.
```

This is one way to fix this, but unsure if we want to go this direction. Feel free to close if not needed.

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [Issue](https://github.com/adobe-private/react-spectrum-v3/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 🧢 Your Project:

React Spectrum

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3462,6 +3462,25 @@
     prop-types "^15.6.1"
     react-lifecycles-compat "^3.0.4"
 
+"@react-spectrum/layout@3.0.0-alpha.1":
```
Comment (snowystinger) [inline yarn.lock:3465]: yarn lock change is wrong, see https://github.com/adobe-private/react-spectrum-v3/pull/640/files#diff-3492513f94746f0f4e50e368715790e4R33 for fix
I don't care if we do it in this PR or in mine, might make more sense in this one

Comment (ross-pfahler) [reply yarn.lock:3465]: all i did was run `yarn`; are we manually editing yarn's output?

Comment (ross-pfahler) [reply yarn.lock:3465]: oh i see you upgraded

Comment (ross-pfahler) [reply yarn.lock:3465]: so im happy if you upgrade first and ill rebase. let me know

Comment (snowystinger) [reply yarn.lock:3465]: yeah, figured we shouldn't have any yarn.lock entries for our own packages ;)

Comment (devongovett) [reply yarn.lock:3465]: why is test utils depending on layout??

Comment (snowystinger) [reply yarn.lock:3465]: o, it used to have a helper function for testing components with slots, however, all tests using it were removed, so we can probably delete that

**Pull request closed without merging**